xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -44,6 +45,9 @@
  */
 abstract class XdsClient {
 
+  // Global counter of active streams with fault injection.
+  final AtomicLong activeFaultInjectedStreams = new AtomicLong();
```

Comment:
Nonono, this is completely wrong. It's anti-pattern for Java. I was thinking about the global atomic should be in some way similar to `SharedCallCounterMap`.

Reply:
`SharedCallCounterMap.getInstance()` is shared per jvm. If you want to share across jvm, then a static AtomicLong field in `XdsNameResolver` would work.